ocaml/xapi/xapi_crashdump.ml

```diff
@@ -11,22 +11,15 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
  *)
-exception Not_implemented
-
-let nothrow f () = try f() with _ -> ()
-
-let create ~__context ~vM ~vDI =
```

Comment:
I've removed it because I would have to add a check to it, but realized it's unused, removing these function saves use time I think, because we don't have to consider them when we make changes.